openpype/tools/publisher/window.py

```diff
@@ -421,6 +431,22 @@ def keyPressEvent(self, event):
         if event.key() == QtCore.Qt.Key_Escape:
             event.accept()
             return
+
+        if event.matches(QtGui.QKeySequence.Save):
+            if not self._controller.publish_has_started:
+                self._save_changes(True)
+            event.accept()
+            return
+
+        if (
+            event.modifiers() == QtCore.Qt.ControlModifier
+            and event.key() == QtCore.Qt.Key_R
+        ):
```

Comment:
If you decide to refactor above to use `QKeySequence.toString()` to get the platform-specific human readable string for it then potentially we might want to do so here too?
```suggestion

        reset_sequence = QtGui.QKeySequence(QtCore.Qt.ControlModifier | 
                                            QtCore.Qt.Key_R)
        if (
            event.matches(reset_sequence)
        ):
```